Riot/Modules/Room/VoiceMessages/VoiceMessagePlaybackController.swift

```diff
@@ -164,8 +164,6 @@ class VoiceMessagePlaybackController: VoiceMessageAudioPlayerDelegate, VoiceMess
         }
         
         self.state = .stopped
-        self.loading = true
```

Comment:
The attachments gets set on the same cell multiple times as it goes through the various sending states and makes everything flicker. There's no good way to stop that from happening, even after comparing event ids, probably because timeline invalidations.